canonical/identity-platform-admin-ui
# WD-15025 - feat: add built verification email

## Changes

Commit the built email template.
NOTE: the logo needs to be attached by the backend. Not sure if that should be done before landing this PR?

https://warthogs.atlassian.net/browse/WD-15025

## QA

- Rebuild the image with this PR (https://github.com/canonical/identity-platform-admin-ui?tab=readme-ov-file#image-rebuild-and-redeployment).
- Create a new identity. You can do this by entering this into the dev console from the UI:
```
fetch("/api/v0/identities", {
  method: "POST",
  body: JSON.stringify({
  schema_id: "default",
  traits: { email: "test123@example.com" },
  credentials: { password: { config: { password: "123" } } },
})
});
```
- Go to the mail UI: http://localhost:18025/ (I had to port forward to access this from outside my multipass: `kubectl port-forward services/mailhog 18025:8025`)
- You should have a "Complete your registration" email that will include the HTML content (without the logo as that needs to be included by the backend).
